CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 1.5.0
```

Comment:
It looks like this can be part of 1.4.0!

We'll need to update the deprecation comments that mention the version too.